tools/engine_tool/lib/src/dependencies.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:process_runner/process_runner.dart';
+
+import 'environment.dart';
+import 'logger.dart';
+
+/// Returns `false` if the Flutter engine's dependencies are up-to-date.
+Future<bool> dependenciesChanged() {
```

Comment:
My 2c: Let's avoid adding code that has no value.

For example, this (and the check in the runner) could just happen in a subsequent PR.

Reply:
Fair enough, removed!